lts.json

```diff
@@ -21,21 +21,21 @@
   "ember-cli-sri": "^2.1.0",
   "ember-cli-test-loader": "^1.1.0",
   "ember-cli-uglify": "^1.2.0",
-  "ember-computed-decorators": "~0.2.0",
-  "ember-concurrency": "~0.7.0",
+  "ember-computed-decorators": "~0.2.2",
+  "ember-concurrency": "~0.7.15",
   "ember-css-modules": "~0.5.0",
   "ember-data": "~2.8.0",
   "ember-data-factory-guy": "^2.0.0",
```

Comment:
I did not test getting both in your project. Both are present because some repos are using Mirage and others are using Factory guy.